packages/atlas/src/components/Table/Table.styles.ts

```diff
@@ -41,6 +42,14 @@ export const Th = styled(Text)`
 export const Td = styled(Text)`
   ${cellStyles};
 
+  overflow: hidden;
+
+  p {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+
```

Comment:
Probably not your change. But I noticed that cells are slightly bigger than in the design, and it's noticeable(at least for me 😅 ). Could you fix this as well? There is too much padding applied to `td`. 
Design: 
![image](https://user-images.githubusercontent.com/51168865/217811490-85d859de-7b73-497d-8f39-2ac654a26ab8.png)
Implementation
![image](https://user-images.githubusercontent.com/51168865/217811778-ed0fd89b-b406-4d11-9642-beaf19f86adb.png)